tests/stress/um/stress_tests_um.cpp

```diff
@@ -0,0 +1,338 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include "bpf/bpf.h"
+#include "bpf/libbpf.h"
+#include "catch_wrapper.hpp"
+#include "ebpf_mt_stress.h"
+#include "program_helper.h"
+#include "test_helper.hpp"
+
+// Data returned by a successful _program_load() call.
+struct free_bpf_object_ptr
+{
+    void
+    operator()(_In_opt_ _Post_invalid_ bpf_object* raw_ptr)
+    {
+        if (raw_ptr != nullptr) {
```

Comment:
Can remove nullptr check to reduce code complexity.

Reply:
fixed